### PR TITLE
Update/integrate last v1 changes

### PR DESCRIPTION
**Pull request by KaiWaldrant**
## Describe your changes
Update to latest changes in openproblems v1

## Checklist before requesting a review
- [x] I have performed a self-review of my code

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [x] New functionality
  - [x] Major changes
  - [ ] Minor changes
  - [ ] Bug fixes

- [ ] Proposed changes are described in the CHANGELOG.md

- [x] CI Tests succeed and look good!

### CODE REVIEW

File: src/tasks/dimensionality_reduction/metrics/rmse/script.py [hunk 1]
```diff
@@ -31,12 +31,13 @@ def _rmse(X, X_emb):
 X_emb = input_embedding.obsm["X_emb"]
 
 print("Compute NNLS residual after SVD", flush=True)
-n_svd = 200
+n_svd = 500
 svd_emb = sklearn.decomposition.TruncatedSVD(n_svd).fit_transform(high_dim)
 rmse = _rmse(svd_emb, X_emb)
 
+#! TODO: change spectral to commit 80b37e7a6aa27df4436f400397564c01276817e0
 print("Compute NLSS residual after spectral embedding", flush=True)
```
Comment (KaiWaldrant) [inline src/tasks/dimensionality_reduction/metrics/rmse/script.py:39]: in openproblems v1 commit 80b37e7a6aa27df4436f400397564c01276817e0 this has changed to use the diffusion map method. @rcannood can you have a look on how to proceed?

**Comment by rcannood** [comment]
> https://github.com/openproblems-bio/openproblems/blob/0a0e902bd1482e35418f7816fc91e9bc31a33126/openproblems/tools/normalize.py changes the cpm normalization to cp10k. (first seen label_projection) 
> @rcannood do we want to make that change as well ?

Makes sense to me!

**Comment by KaiWaldrant** [comment]
just to be sure recreate and upload the test resources
